docs/build.md

```diff
@@ -0,0 +1,128 @@
+# Build instructions
+
+The pipeline requires Python version 3.5.
+
+## Python 3.5 installation (optional)
+The instructions below will be useful:
+* In case you have a newer version and need to run the pipeline you can follow the next steps to install Python 3.5
+without replacing you default `python`;
+* If you are running the pipeline on the `ebi-cli` cluster, which currently only supports Python 3.4.
+
+Adjust `VERSION` and `INST_PATH` if needed.
```

Comment:
`INSTALL_PATH` or `INSTALLATION_PATH` for increased readability.

Reply:
Replaced with `INSTALL_PATH`